examples/github-appcards/src/components/Miro/NoSelection.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from "react";
+import selection from "../../assets/selection.png";
+
+const NoSelection = () => {
+  return (
+    <>
+      <h3>Select from Miro Board</h3>
+      <p>
+        Select at least one sticky note or Miro card from the board to convert
+        to a GitHub card.
```

Comment:
Suggestion:

"From the board, select at least a sticky note or a card to convert it to a GitHub card."